### PR TITLE
New package: jalv-1.6.6

### PR DESCRIPTION
**Pull request by tibequadorian**
I've only tested this on x86_64-glibc

#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/jalv/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'jalv'
+pkgname=jalv
+version=1.6.6
+revision=1
+build_style=waf3
+configure_args="$(vopt_if portaudio '--portaudio') --no-gtkmm
+ $(vopt_if gtk '' '--no-gtk2') $(vopt_if gtk3 '' '--no-gtk3')
```
Comment (ericonr) [inline srcpkgs/jalv/template:7]: `gtk2` shouldn't be made available at all.

Comment (tibequadorian) [reply srcpkgs/jalv/template]: okay I removed it

Comment (tibequadorian) [reply srcpkgs/jalv/template]: oops okay now I fully removed it

**Comment by tibequadorian** [comment]
@ericonr is it ready to merge?

File: srcpkgs/jalv/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+build_style=waf3
+configure_args="$(vopt_if portaudio '--portaudio') --no-gtkmm --no-gtk2
+ $(vopt_if gtk3 '' '--no-gtk3') $(vopt_if qt '' '--no-qt5')"
+hostmakedepends="pkg-config $(vopt_if qt qt5-host-tools)"
+makedepends="lv2 lilv-devel suil-devel serd-devel sord-devel sratom-devel
+ $(vopt_if portaudio portaudio-devel jack-devel) $(vopt_if gtk3 gtk+3-devel)
+ $(vopt_if qt qt5-devel)"
+short_desc="LV2 host for JACK"
+maintainer="tibequadorian <tibequadorian@posteo.de>"
+license="GPL-3.0-or-later"
```
Comment (ericonr) [inline srcpkgs/jalv/template:14]: Should be ISC https://gitlab.com/drobilla/jalv/-/blob/master/COPYING

Comment (tibequadorian) [reply srcpkgs/jalv/template:14]: oh :sweat_smile: sorry

**Pull request merged**